src/homepageExperience/components/steps/cli/ExecuteQuery.tsx

```diff
@@ -8,8 +8,15 @@ const logCopyCodeSnippet = () => {
   event('firstMile.cliWizard.executeQuery.code.copied')
 }
 
-export const ExecuteQuery = () => {
-  const query = `influx query 'from(bucket:"sample-bucket") |> range(start:-10m)' --raw`
+type OwnProps = {
+  bucket: string
+}
+
+export const ExecuteQuery = (props: OwnProps) => {
+  const {bucket} = props
+  const query = `influx query 'from(bucket:"${
+    bucket === '<BUCKET>' ? 'sample-bucket' : bucket
+  }") |> range(start:-10m)' --raw`
```

Comment:
it'd probably be cleaner and easier to follow if this expression to get the proper bucket name was stored into a variable.

```ts
const bucketName = bucket === '<BUCKET>' ? 'sample-bucket' : bucket
const query = `influx query 'from(bucket:"${bucketName}") |> range(start:-10m)' --raw`
```